lib/apiservers/engine/backends/container.go

```diff
@@ -385,6 +385,37 @@ func (c *Container) ContainerExecStart(ctx context.Context, eid string, stdin io
 	actor := CreateContainerEventActorWithAttributes(vc, map[string]string{})
 	EventService().Log(event, eventtypes.ContainerEventType, actor)
 
+	handle, err = c.Handle(id, name)
+	if err != nil {
+		return err
```

Comment:
This should `return InternalServerError(err.Error())` to be consistent with the task.Bind code.